tests/resources/postgresql_test_tables.sql

```diff
@@ -805,6 +805,38 @@ INSERT INTO pso_data_validator.dvt_group_by_timestamp VALUES
 (5,2,DATE'2022-02-02',TIMESTAMP'2022-02-02 13:00:00'),
 (6,3,DATE'2023-03-03',TIMESTAMP'2023-03-03 12:00:00');
 
+DROP TABLE pso_data_validator.dvt_high_epoch_seconds;
```

Comment:
```suggestion
DROP TABLE IF EXISTS pso_data_validator.dvt_high_epoch_seconds;
```
When executing this .SQL file on my local env I had to add this to every DROP statement to be able to have an execution without errors. I'll create a draft PR already with the adjustments I've been doing to the test tables scripts. Meanwhile it'd be great to have new tables being added already with the IF EXISTS part.

Reply:
Draft PR opened, Postgres file here: https://github.com/GoogleCloudPlatform/professional-services-data-validator/pull/1416/files#diff-43590f1f4fd814d6c014c7819fb1769f24e67238b0f4d65c2523ed447f13c72d